mob_test.go

```diff
@@ -1505,6 +1506,56 @@ func TestHelpRequested(t *testing.T) {
 	equals(t, true, helpRequested([]string{"s", "10", "-h"}))
 }
 
+func TestAbortTimerIfNewTimerIsStarted(t *testing.T) {
+	_, configuration := setup(t)
+	startTimer("10", configuration)
+	time.Sleep(time.Millisecond)
```

Comment:
Oh really for me it worked. We could try a second. I‘m not sure if we want to add a library for this. As our design principle says we do not want to add libraries because we need to have them up to date. Even it is just Test code I think a sleep for a second would be fine or we could check if we find an easy solution that we can implement ourself.

Reply:
I agree. A second is a bit long though